Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -42,9 +42,43 @@ public enum NIOSSLPrivateKeySource {
 
 /// Places NIOSSL can obtain a trust store from.
 public enum NIOSSLTrustRoots {
+    /// Path to either a file of CA certificates in PEM format, or a directory containing CA certificates in PEM format.
+    ///
+    /// If a path to a file is provided, the file can contain several CA certificates identified by
+    ///
+    ///     -----BEGIN CERTIFICATE-----
+    ///     ... (CA certificate in base64 encoding) ...
+    ///     -----END CERTIFICATE-----
+    ///
+    /// sequences. Before, between, and after the certificates, text is allowed which can be used e.g.
+    /// for descriptions of the certificates.
+    ///
+    /// If a path to a directory is provided, the files each contain one CA certificate in PEM format.
     case file(String)
```

Comment:
When used in the base trust root system this actually doesn't have to be a PEM file, it can be a DER file as well: we pass it directly to the load functions.

Reply:
Similarly when pointing at a directory those certs do not have to be in PEM format.

Reply:
Oh, interesting. FWICT `NIOSSLContext.configureCertificateValidation` calls  `NIOSSLContext.loadVerifyLocations` when passed `.file(path)` which just calls through to `SSL_CTX_load_verify_locations` with the path provided.

The documentation I provided on this enum case was based on the man page which states:
> If CAfile is not NULL, it points to a file of CA certificates in PEM format.

and later on

> If CApath is not NULL, it points to a directory containing CA certificates in PEM format.

Link: https://www.openssl.org/docs/man1.0.2/man3/SSL_CTX_load_verify_locations.html

Reply:
Yup, you're right. Let's leave the comment as is then. Good catch!